core/src/commonMain/kotlin/entity/interaction/Interaction.kt

```diff
@@ -24,10 +25,13 @@ public sealed interface Interaction : InteractionBehavior {
 
     override val applicationId: Snowflake get() = data.applicationId
 
-    override val channelId: Snowflake get() = data.channelId
+    override val channelId: Snowflake get() = data.channelId.value!!
```

Comment:
hm, i think this should be made nullable instead. i also think it should use the id from the `channel` field (["We recommend that you begin using this channel field to identify the source of channel of the interaction [...]"](https://discord.com/developers/docs/change-log#interaction-channel-data))